src/Main/Environment.hs

```diff
@@ -126,20 +127,28 @@ testProcess check defaultMsg testName prog args inp maudeTest = do
 ensureGraphVizDot :: Arguments -> IO Bool
 ensureGraphVizDot as = do
     putStrLn $ "GraphViz tool: '" ++ dot ++ "'"
-    testProcess check errMsg " checking version: " dot ["-V"] "" False
+    testProcess check errMsg1 " checking version: " dot ["-v </dev/null"] "" False
```

Comment:
Is this construction reliable cross-platform? I'm not sure how this would work for a future windows version.

Reply:
Reading the graphviz documentation at https://graphviz.org/doc/info/output.html , it is suggested that the output formats should be listed by running `dot -T?`. I would suggest we use that instead, since it is recommended in the docs (and avoids the cross platform issue).

Reply:
Ah I totally forgot about windows. Yes, it seems like a better approach, which I use it now.